client/index.jsx

```diff
@@ -17,10 +21,13 @@ class App extends React.Component {
   };
 
   getData() {
-    Axios.get('/hi')
+    var parts = document.URL.split("/");
+    var lastSegment = parts.pop() || parts.pop();
```

Comment:
The `OR` statement looks interesting here, looks like they are the same functionality?